products/developertoolset/get-started-rhel7-cpp.adoc

```diff
@@ -1,7 +1,15 @@
 :awestruct-layout: product-get-started-multipath
 :awestruct-interpolate: true
-:title: "Developer Toolset - Get started developing with C++ and Eclipse on RHEL 7"
-:awestruct-description: "Get started developing with C++ and Eclipse from Red Hat Developer Toolset on Red Hat Enterprise Linux 7"
+:tthw-site-base-url: #{site.base_url}
+:tthw-cdn-site-base-url: #{cdn(site.base_url)}
```

Comment:
Looks like for whatever reason things are being encoded and not resolved. No idea why this is happening. Can you do without the images for now?